### PR TITLE
3.9 release post

### PR DESCRIPTION
**Pull request by kennethlarsen**
# Currently waiting for the release of Data and CLI

# Framework
No new features.

8 bugfixes

# CLI
- [x] Update ember-welcome-page usage to angle brackets: https://github.com/ember-cli/ember-cli/pull/8383

# Data
- [x] Replace jQuery with fetch: https://github.com/emberjs/data/pull/5900

### CODE REVIEW

File: source/2019-04-10-ember-3-9-released.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
+---
+title: Ember 3.9 Released
+author: Kenneth Larsen
+tags: Releases, 2018, 3, 3.9
```
Comment (amyrlam) [inline source/2019-04-10-ember-3-9-released.md]:
```suggestion
tags: Releases, 2019, 3, 3.9
```

Got this from https://github.com/ember-learn/ember-blog/pull/37

File: source/2019-04-10-ember-3-9-released.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+responsive: true
+---
+
+Today the Ember project is releasing version 3.9 of Ember.js, Ember Data, and Ember CLI. This release kicks off the 3.5 beta cycle for all sub-projects. We encourage our community (especially addon authors) to help test these beta builds and report any bugs before they are published as a final release in six weeks' time. The [ember-try](https://github.com/ember-cli/ember-try) addon is a great way to continuously test your projects against the latest Ember releases.
```
Comment (amyrlam) [inline source/2019-04-10-ember-3-9-released.md]:
```suggestion
Today the Ember project is releasing version 3.9 of Ember.js, Ember Data, and Ember CLI. This release kicks off the 3.10 beta cycle for all sub-projects. We encourage our community (especially addon authors) to help test these beta builds and report any bugs before they are published as a final release in six weeks' time. The [ember-try](https://github.com/ember-cli/ember-try) addon is a great way to continuously test your projects against the latest Ember releases.
```

**Comment by kennethlarsen** [comment]
omg thank you @amyrlam. Need to update the template for these posts

File: source/2019-04-10-ember-3-9-released.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+`.property()` is a modifier that adds additional property dependencies to an existing computed property. To update, move the dependencies to the main computed property definition and you shouldn't see a deprecation warning any more. For more information please refer to [the deprecations app](https://emberjs.com/deprecations/v3.x#toc_computed-property-property).
+
+**Computed Property Volatility (3 of 4)**
+
+`.volatile()` is a computed property modifier which makes a computed property recalculate every time it is accessed, instead of caching. It also prevents property notifications from ever occuring on the property, which is generally not the behavior that developers are after. Volatile properties are usually used to simulate the behavior of native getters, which means that they would otherwise behave like normal properties.
+
+To update, consider upgrading to native class syntax and using native getters directly instead. There's guide on how to do this in the [deprecations app](https://emberjs.com/deprecations/v3.x#toc_computed-property-volatile).
+
+**Deprecate `@ember/object#aliasMethod` (4 of 4)
```
Comment (kategengler) [inline source/2019-04-10-ember-3-9-released.md]: Missing end ** here; formatting when rendered is incorrect

Comment (kennethlarsen) [reply source/2019-04-10-ember-3-9-released.md]: Thanks!

File: source/2019-04-10-ember-3-9-released.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+- [Ember LTS Releases](http://emberjs.com/blog/2016/02/25/announcing-embers-first-lts.html)
+
+---
+
+## Ember.js
+
+Ember.js is the core framework for building ambitious web applications.
+
+### Changes in Ember.js 3.9
+Ember.js 3.9 is an incremental, backwards compatible release of Ember with bugfixes, performance improvements, and minor deprecations. There are zero (0) new features, four (4) deprecations, and six (6) bugfixes in this version.
```
Comment (chadhietala) [inline source/2019-04-10-ember-3-9-released.md]: I think there were 8 bugfixs: https://github.com/emberjs/ember.js/blob/master/CHANGELOG.md#v390-april-01-2019

Updating router_js was a bug fix.

**Comment by kategengler** [comment]
Is it worth mentioning that 3.8 is now promoted to LTS?

File: source/2019-04-10-ember-3-9-released.md [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+
+#### New Features (0)
+No new features introduced in Ember.js 3.9.
+
+#### Deprecations (4)
```
Comment (pzuraq) [inline source/2019-04-10-ember-3-9-released.md:31]: Deprecation messages look good 👍

File: source/2019-04-09-ember-3-9-released.md [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+
+You can read more about our general release process here:
+
+- [Release Dashboard](http://emberjs.com/builds/)
+- [The Ember Release Cycle](http://emberjs.com/blog/2013/09/06/new-ember-release-process.html)
+- [The Ember Project](http://emberjs.com/blog/2015/06/16/ember-project-at-2-0.html)
+- [Ember LTS Releases](http://emberjs.com/blog/2016/02/25/announcing-embers-first-lts.html)
+
+Additionally, versions 3.9 of Ember and Ember Data are now promoted to LTS, which stands for Long Term Support. An LTS version of Ember continues to receive security updates for 9 release cycles (54 weeks) and bugfixes for 6 cycles (36 weeks). LTS releases typically occur every four minor versions. The previous LTS version for Ember was 3.4.
```
Comment (Panman82) [inline source/2019-04-09-ember-3-9-released.md]: I think it should be 3.8 for LTS, not 3.9.?.

Comment (kennethlarsen) [reply source/2019-04-09-ember-3-9-released.md]: Oh, that was a brainfart. Thank you so much, @Panman8201 ✨

**Comment by kennethlarsen** [comment]
CLI update: "noteworthy things for CLI: Node 11 support and MU blueprint updates"

File: source/2019-04-09-ember-3-9-released.md [hunk 7]
```diff
@@ -0,0 +1,132 @@
...
+
+#### New Features (2)
+
+**`ember-welcome-page` updated to use angle bracket invocation syntax (1 of 2)**
+
+The welcome page that is generated in new Ember apps is now converted to use the angle bracket invocation syntax. This means that instead of adding `{{welcome-page}}` to `application.hbs` we now add `<WelcomePage />`.
+
+**Support for Node 11 (2 of 2)**
+
+Ember CLI is now tested against Node 11. If developers use it for their apps and addons, the CLI will not show a warning.
```
Comment (Turbo87) [inline source/2019-04-09-ember-3-9-released.md]:
```suggestion
Ember CLI is now tested against Node 11. If developers use it for their apps and addons, the CLI will not show a warning anymore.
```

File: source/2019-04-09-ember-3-9-released.md [hunk 8]
```diff
@@ -0,0 +1,132 @@
...
+
+#### Bugfixes (1)
+
+**Module unification blueprint updates (1 of 1)**
+
+The MU blueprints were not enabling the `EMBER_MODULE_UNIFICATION` Ember Feature Flag due to flag capitalization variations. This has been fixed.
```
Comment (Turbo87) [inline source/2019-04-09-ember-3-9-released.md]:
```suggestion
The "Module Unification" blueprints have received several bugfixes and enhancements. For more details have a look at https://github.com/ember-cli/ember-cli/releases/tag/v3.9.0.
```

**Review by jenweber — APPROVED** [review]

**Comment by simonihmig** [comment]
I just read the blog post, and noticed that it does not mention the deprecations (2) around Ember's jQuery integration (RFC386), which I think a *lot* of apps/addons will run into. So should be mentioned IMHO!

FWIW, they do show up in the [Changelog](https://github.com/emberjs/ember.js/blob/master/CHANGELOG.md#v390-april-01-2019)

**Comment by kennethlarsen** [comment]
@simonihmig Thank you! I didn't realize that was in 3.9, will add it.

**Pull request merged**